mdpow/workflows/dihedrals.py

```diff
@@ -573,43 +615,52 @@ def automated_dihedral_analysis(dirname=None, df_save_dir=None, figdir=None,
            available.
 
        *df_save_dir*
-           path to the location to save results :ref:`pandas.DataFrame <https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html>`
+           optional, path to the location to save results :class:`pandas.DataFrame`
 
        *figdir*
-           optional, path to an existing directory where plots
-           can be saved for each dihedral atom group
+           optional, path to the location to save figures
 
        *resname*
-           resname for the molecule as defined in 
+           `resname` for the molecule as defined in 
            the topology and trajectory
 
        *molname*
            molecule name to be used for labelling
-           plots, if different from resname
+           plots, if different from `resname`
            
        *SMARTS*
            The default SMARTS string is described in detail under :data:`SMARTS_DEFAULT`.
 
-       *dataframe : Pandas DataFrame*
+       *dataframe*
            optional, if :class:`~mdpow.analysis.dihedral.DihedralAnalysis`
            was done prior, then results :class:`pandas.DataFrame` can be
            input to utilize angle padding and violin plotting functionality
 
        *padding*
            value in degrees
            default: 45
-           see :func:`~mdpow.workflows.dihedrals.periodic_angle`
+       .. seealso:: :func:`~mdpow.workflows.dihedrals.periodic_angle`
```

Comment:
indent to be in the same block as the rest of the padding text, otherwise breaks flow